com.playeveryware.eos/Runtime/Core/Config/ProductionEnvironments.cs

```diff
@@ -50,6 +50,33 @@ public ProductionEnvironments()
             Sandboxes.SetRemovePredicate(CanSandboxBeRemoved);
         }
 
+        /// <summary>
+        /// Tries to retrieve the first defined named deployment.
+        /// </summary>
+        /// <param name="deployment">
+        /// The first named deployment that has been determined to be defined.
+        /// </param>
+        /// <returns>
+        /// True if there was a defined named deployment found, false otherwise.
+        /// </returns>
+        public bool TryGetFirstDefinedNamedDeployment(out Named<Deployment> deployment)
+        {
+            deployment = null;
+
+            // Go through the deployments
+            foreach (var dep in Deployments)
+            {
+                // If the deployment is complete then stop
+                if (dep.Value.IsComplete)
+                {
+                    deployment = dep;
+                    break;
+                }
+            }
+
+            return (deployment == null);
```

Comment:
This has a boolean logic error; I think it should return true if deployment is not null.

Reply:
Good catch - I've corrected it.